test/wpt/test-webcrypto.js

```diff
@@ -9,7 +9,7 @@ const { WPTRunner } = require('../common/wpt');
 const runner = new WPTRunner('WebCryptoAPI');
 
 runner.setInitScript(`
-  global.location = {};
+  globalThis.location ||= {};
```

Comment:
What was the reason for this change? I'm curious

Reply:
First of all to align the use of globalThis over global. Second so that this is a noop if we ever added a location global.